models/migrations/v214.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"xorm.io/xorm"
+)
+
+func addAutoMergeTable(x *xorm.Engine) error {
+	type MergeStyle string
+	type PullAutoMerge struct {
+		ID          int64      `xorm:"pk autoincr"`
+		PullID      int64      `xorm:"BIGINT"`
+		DoerID      int64      `xorm:"BIGINT"`
+		MergeStyle  MergeStyle `xorm:"varchar(50)"`
+		Message     string     `xorm:"TEXT"`
+		CreatedUnix int64      `xorm:"created"`
+	}
```

Comment:
1. No `index` ?
2. `` 	Message     string     `xorm:"TEXT"` `` is not friendly to MySQL. It sould be `LONGTEXT, otherwise only 64K can be stored.